cppcache/include/geode/DataInput.hpp

```diff
@@ -502,7 +502,7 @@ class APACHE_GEODE_EXPORT DataInput {
 
   inline char readPdxChar() { return static_cast<char>(readInt16()); }
 
-  inline void _checkBufferSize(size_t size, int32_t line) {
+  virtual inline void _checkBufferSize(size_t size, int32_t line) {
```

Comment:
Having a mix of inline and virtual attributes produces some really interesting unknown behavior.

Reply:
Haha! You would think.